server/models/boxCrime.js

```diff
@@ -0,0 +1,104 @@
+var db = require('../../database/index').db;
+var BoxCrime = require('../../database/models/BoxCrime');
+
+var storeOpenDataInBoxes = (crimeData) => {
+  
+  var asyncBoxStore = crimeData.forEach((crime) => {
+    return new Promise((res, rej) => {
+
+      var long = crime.location.longitude - 0;
+      var lat = crime.location.latitude - 0;
+      var mutilplier = 1000000000000;
+      var boxPadding = 20000000;
+      
+      var coords = {
+        upperLeft: [(((long * mutilplier) - boxPadding) / mutilplier), (((lat * mutilplier) + boxPadding) / mutilplier)],
```

Comment:
Yeah, I was trying to avoid the whole "javascript doesn't like to add decimals properly" thing, since I'm adding a number 7 places past the decimal. The multiplier and boxPadding variables don't have to be such big numbers, so long as we keep them proportional and to the right of the decimal place (except for fine tuning the actual size of the box, of course).